frontend/src/app/testQueue/addToQueue/SearchInput.jsx

```diff
@@ -12,7 +19,7 @@ const SearchInput = ({
   return (
     <React.Fragment>
       <TextInput
-        placeholder="Search by Unique Patient ID or Name"
+        placeholder={"Search by Unique " + PATIENT_TERM_CAP + "ID or Name"}
```

Comment:
So I guess this should read `PersonID` then? Just a suggestion here to use backticks so the lack of a space there will be more visible:
```suggestion
        placeholder={`Search by Unique ${PATIENT_TERM_CAP}ID or Name`}
```